### ISSUE TITLE
Table row height auto scale

### ISSUE THREAD

**Issue by DJ-Raven** [JFormDesigner/FlatLaf#682]
Table row height don't auto scale when I put flat client properties style. Is this possible to fix this ? Thank you.
```java
table.putClientProperty(FlatClientProperties.STYLE, ""
                + "rowHeight:70");
```
Then I call
```java
UIManager.put("defaultFont", newFontWithSize);
```

**Comment by DevCharly** [comment]
Thanks for reporting 👍 

`rowHeight` for  Tree and`iconTextGap` for Button, CheckBox and RadioButton had the same problem

fixed in latest `3.2-SNAPSHOT`: https://github.com/JFormDesigner/FlatLaf#snapshots

**Comment by DJ-Raven** [comment]
Thank you
